### PR TITLE
casestudypart1

### PR DESCRIPTION
**Pull request by abioduna**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    </head>
+    <body>
+    <header><h1>Pacific Trails Resort</h1></header>
+    <nav>
+        <a href="index.html">Home</a> &nbsp;
+        <a href="yurts.html">Yurts</a> &nbsp;
+        <a href="activities">Activities</a> &nbsp;
+        <a href="reservations">Reservations</a> &nbsp;     
+    </nav>
+    <div id="content">
```
Comment (mbMosman) [inline index.html]: We're going all HTML5 here, so change this to `<main>`.  A late update to the instructions for the new edition of the textbook.  Make sure to update the closing tag too.

File: yurts.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    </head>
+    <body>
+    <header><h1>Pacific Trails Resort</h1></header>
+    <nav>
+        <a href="index.html">Home</a> &nbsp;
+        <a href="yurts.html">Yurts</a> &nbsp;
+        <a href="activities">Activities</a> &nbsp;
+        <a href="reservations">Reservations</a> &nbsp;     
+    </nav>
+    <div id="content">
```
Comment (mbMosman) [inline yurts.html]: We're going all HTML5 here, so change this to `<main>`.  A late update to the instructions for the new edition of the textbook.  Make sure to update the closing tag too.

File: yurts.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (mbMosman) [inline yurts.html]: It'd help w/ the grading if you would use the Brackets beautify plugin to format your code before committing.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        <li>Activities lodge with fireplace and gift shop</li>
+        <li>Nightly fine dining at the Overlook Cafe</li>
+        <li>Heated outdoor pool and whirlpool</li>
+        <li>Guided Hiking tours of the redwoods</li>
+        </ul>
+        <div>
+            Pacific Trails Resort<br>
+            12010 Pacific Trails Road<br>
+            Zephyr, CA 95555<br>
+
```
Comment (mbMosman) [inline index.html]: You're missing a blank line here before the phone number.  You can get that by adding an additional `br`

File: pacific.css [hunk 5]
```diff
@@ -0,0 +1,6 @@
+body { background-color: #ffffff; }
+    h1 { background-color: #000033;
```
Comment (mbMosman) [inline pacific.css]: The background and font colors should be set for the `<header>` element, instead of the `<h1>`

**Comment by mbMosman** [comment]
Good job!  Looks great.

**Comment by abioduna** [comment]
I have been having a hard time finding beautify on brackets. How do i do that?

**Pull request closed without merging**